src/js/components/Ballot/MeasureItemCompressed.jsx

```diff
@@ -252,12 +256,12 @@ export default class MeasureItemCompressed extends Component {
     ) :
       null;
 
-    const comment_display_raccoon_mobile = this.props.showPositionStatementActionBar || is_voter_support || is_voter_oppose || voter_statement_text ? (
+    const commentDisplayRaccoonMobile = this.props.showPositionStatementActionBar || isVoterSupport || isVoterOppose || voterStatementText ? (
       <div className="d-block d-sm-none o-media-object u-flex-auto u-min-50 u-push--sm u-stack--sm">
         <div className="o-media-object__body u-flex u-flex-column u-flex-auto u-justify-between">
           <ItemPositionStatementActionBar
             ballot_item_we_vote_id={this.state.ballotItemWeVoteId}
-            ballot_item_display_name={this.state.ballot_item_display_name}
+            ballotItemDisplayName={this.state.ballotItemDisplayName}
```

Comment:
To make this change you would need to update ItemPositionStatementActionBar since the prop is still `ballot_item_display_name`.